File: trunk/CoverageTest/InterwayDocs/Plain-CS/Codisa.InterwayDocs.Business/DatabaseInterwayDocs.cs
using System.Configuration;

namespace Codisa.InterwayDocs.Business
{
    /// <summary>Class that provides the connection
    /// strings used by the application.</summary>
    internal static class Database
    {
        /// <summary>Connection string to the InterwayDocs database.</summary>
        internal static string InterwayDocsConnection
        {
            get { return ConfigurationManager.ConnectionStrings["InterwayDocs"].ConnectionString; }
        }
    }
}

File: trunk/CoverageTest/InterwayDocs/Plain-VB/Codisa.InterwayDocs.Business/DatabaseInterwayDocs.vb
Imports System.Configuration

Namespace Codisa.InterwayDocs.Business

    ''' <summary>Class that provides the connection
    ''' strings used by the application.</summary>
    Friend Class Database

        ''' <summary>Connection string to the InterwayDocs database.</summary>
        Friend Shared ReadOnly Property InterwayDocsConnection As String
            Get
                Return ConfigurationManager.ConnectionStrings("InterwayDocs").ConnectionString
            End Get
        End Property

    End Class

End Namespace
